terraform/modules/sns/sns.tf

```
@@ -0,0 +1,19 @@
+resource "aws_sns_topic" "cg_notifications" {
+  name = var.sns_main_name
+}
+
+resource "aws_sns_topic" "cg_platform_notifications" {
+  name = var.sns_slack_name
+}
+
+resource "aws_sns_topic_subscription" "cg_notifications" {
+  topic_arn = aws_sns_topic.cg_notifications.arn
+  protocol  = "email"
+  endpoint  = var.sns_main_endpoint
+}
+
+resource "aws_sns_topic_subscription" "cg_platform_notifications" {
```

```suggestion
resource "aws_sns_topic_subscription" "cg_platform_slack_notifications" {
```

Just a suggestion to make it clear that these notifications go to Slack